### PR TITLE
RTP packetization editorial changes

### PR DESCRIPTION
**Pull request by murillo128**
@youennf PTAL

### CODE REVIEW

File: codec-agnostic-rtp-payload-format.md [hunk 1]
```diff
@@ -44,23 +45,24 @@ informative:
...
+It is the responsibility of the application using the packetizer to group media content in meaningful frames. In the common case of a video codec, the packetizer frame is the frame in byte format (h264 annex b for example) generated by the encoder. 
 
-In the case of a video codec supporting spatial scalability, each spatial layer is split in its own frame.
-The spatial layers are sent in ascending order, with the same RTP timestamp, and only the last RTP packet of the last spatial layer frame will have the marker bit set to 1.
+When the packetizer receives a frame from the application, it MUST fragment the frame content in multiple RTP packets to ensure packets do not exceed the network maximum transmission unit. The content of the frame will be treated as a binary blob by the packetizer, so the decision about the boundaries of each fragment is decided arbitrarily by the packetizer. The packetizer MUST NOT modify the frame content and the concatenating the RTP payload of the RTP packets for each frame MUST produce the exact binary content of the input frame content.
```
Comment (youennf) [inline codec-agnostic-rtp-payload-format.md]: I am not sure what we cover with 'The packetizer MUST NOT modify the frame content '.
Typo: s/the concatenating/concatenating/

File: codec-agnostic-rtp-payload-format.md [hunk 2]
```diff
@@ -44,23 +45,24 @@ informative:
...
+
+The marker bit of each RTP packet in a frame MUST be set according to the audio and video profiles specified in [[rfc3551]].
+
+In the case of a video codec supporting spatial scalability, each spatial layer MUST be split in its own frame by the application before passing it to the packetizer. 
+The spatial layer frames are sent in ascending order, with the same RTP timestamp, and only the last RTP packet of the last spatial layer frame will have the marker bit set to 1.
+
+If the application wants to encrypt content, the application needs to split media content into frames before encryption as explained above and, using {{SFrame}} for example, encrypt each frame content independently. The content of each encrypted frame will then be fed to the packetizer.
```
Comment (youennf) [inline codec-agnostic-rtp-payload-format.md]: Maybe these last two sentences should go next to the sentence starting with "It is the responsibility of the application using the packetizer to group media content in meaningful frames".
It makes it clear why it is the responsibility of the application to do so

**Review by youennf — APPROVED** [review]

File: codec-agnostic-rtp-payload-format.md (hunk 1 above)
Comment (murillo128) [reply codec-agnostic-rtp-payload-format.md]: I have added also that relying servers must not modify the content as well.

File: codec-agnostic-rtp-payload-format.md (hunk 2 above)
Comment (murillo128) [reply codec-agnostic-rtp-payload-format.md]: Moved the two sentences related to the app close to the first mention of the app.

**Comment by youennf** [comment]
Can we merge this PR?
Even if we are not sure about how to handle RED (within payload or as header extension), this PR adds some nice improvements.

**Pull request merged**